### PR TITLE
docs(fileinput): standalone page for FileInput component

### PR DESCRIPTION
**Pull request by MateoWartelle**
- [X] I have followed the [Your First Code Contribution section of the Contributing guide](https://github.com/themesberg/flowbite-react/blob/main/CONTRIBUTING.md#your-first-code-contribution)

Summarize the changes made and the motivation behind them.

To create separate pages for /forms/* documentations to match flowbite of existing react components. 

Reference related issues using `#` followed by the issue number.

If there are breaking API changes - like adding or removing props, or changing the structure of the theme - describe them, and provide steps to update existing code.

### CODE REVIEW

File: content/docs/forms/file-input.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+```jsx
+import { FileInput } from 'flowbite-react';
+```
+
+## File upload example
+
+Get started with a simple `<FileInput>` component to let users upload one single file.
+
+<Example name="fileInput.example" />
```
Comment (SutuSebastian) [inline content/docs/forms/file-input.mdx:20]: to keep the pattern should we rename to `fileInput.root`? 🤔

**Review by SutuSebastian — APPROVED** [review]
LGTM

File: content/docs/forms/file-input.mdx (hunk 1 above)
Comment (MateoWartelle) [reply content/docs/forms/file-input.mdx]: Yes I will change.

**Comment by MateoWartelle** [comment]
Ill rebase and it should be good to merge.

**Comment by MateoWartelle** [comment]
ok should be good to go.

thanks

**Comment by rluders** [comment]
@MateoWartelle don't we want to remove it from here? https://flowbite-react-git-fork-mateowartelle-docs-fileinput-themesberg.vercel.app/docs/components/forms#file-upload

**Comment by SutuSebastian** [comment]
> @MateoWartelle don't we want to remove it from here? https://flowbite-react-git-fork-mateowartelle-docs-fileinput-themesberg.vercel.app/docs/components/forms#file-upload

I actually discussed this subject with @zoltanszogyenyi a couple of days ago and we want to keep the `components/forms/*` page as an overall form components examples, and each form component should have its own page under `forms/*` with even more detailed implementation and examples.

**Pull request merged**